addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -698,4 +685,29 @@ private static String insertUninitVar(String cmd) {
         return cmd.replaceAll("\\s", Matcher.quoteReplacement(var + " "))
                 .replaceAll("\\/", Matcher.quoteReplacement(var + "/"));
     }
+
+    private AlertBuilder buildAlert(
+            String param, String attack, String evidence, String otherInfo, HttpMessage msg) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence)
+                .setMessage(msg)
+                .setOtherInfo(otherInfo);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(
+                buildAlert(
+                                "qry",
+                                "a;cat /etc/passwd ",
+                                "root:x:0:0",
+                                "The scan rule was able to retrieve the content of a file or "
+                                        + "command by sending [a;cat /etc/passwd ] to the operating "
+                                        + "system running this application",
```

Comment:
Should use the resource message, otherwise we'll have to manually maintain it (most likely it will go out of sync).